test/test-execute/exec-temporaryfilesystem-ro.service

```diff
@@ -10,6 +10,9 @@ ExecStart=/bin/sh -c 'test -d /var/test-exec-temporaryfilesystem/rw && test -d /
 # Check TemporaryFileSystem= are empty
 ExecStart=/bin/sh -c 'for i in $$(ls -A /var); do test $$i = test-exec-temporaryfilesystem || false; done'
 
+# Check default mode
+ExecStart=sh -x -c '[ "$(stat -c %%a /var)" == 755 ]'
```

Comment:
```
Executing: /bin/sh -x -c '[ "$(stat -c %a /var)" == 755 ]'
+ stat -c %a /var
+ [ 755 == 755 ]
/bin/sh: 1: [: 755: unexpected operator
```
We usually use `=` instead of `==`. Also, in other tests, we use `test` instead of `[`.